org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/tracking/CombinedTrackable.java

```diff
@@ -37,6 +39,10 @@ public CombinedTrackable(Trackable serverIssue, Trackable currentIssue) {
     this.creationDate = serverIssue.getCreationDate();
     this.resolved = serverIssue.isResolved();
     this.assignee = serverIssue.getAssignee();
+    this.severity = serverIssue.getSeverity();
+    // TODO when we are able to get server side issue type
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)